sklearn/preprocessing/_encoders.py

```diff
@@ -70,19 +71,36 @@ def _get_feature(self, X, feature_idx):
         # numpy arrays, sparse arrays
         return X[:, feature_idx]
 
-    def _fit(self, X, handle_unknown='error'):
+    def _fit(self, X, handle_unknown='error', ordinal=False):
```

Comment:
IMO, `encoding='ordinal'` or `encoding='one-hot'` would be better than a boolean, even more, if we will have some new encoder in the future.